modules/gitops-release-notes-1-9-2.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-9-2_{context}"]
+= Release notes for {gitops-title} 1.9.2
+
+{gitops-title} 1.9.2 is now available on {product-title} 4.12 and 4.13.
+
+[id="errata-updates-1-9-2_{context}"]
+== Errata updates
+
+=== RHSA-2023:5029 - {gitops-title} 1.9.2 security update advisory
```

Comment:
```suggestion
[id="errata-security-update-advisory-1-9-2_{context}"]
=== RHSA-2023:5029 - {gitops-title} 1.9.2 security update advisory
```

Reply:
@masauso-lungu Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

- All titles and section headings must have an anchor ID. The anchor ID must be similar to the title or section heading.
- You must add the {context} variable to the end of each anchor ID in module files.

Reply:
what about what is in line 12?

Reply:
@masauso-lungu Check the anchor id change now.